src/Services/Qdm/Services/LaboratoryTestService.php

```diff
@@ -63,6 +67,16 @@ public function makeQdmModel(QdmRecord $recordObj)
             'relevantDatetime' => new DateTime([
                 'date' => $record['date']
             ]),
+            'relevantPeriod' => new Interval([
+                'low' => new DateTime([
+                    'date' => $record['date']
+                ]),
+                'high' => new DateTime([
+                    'date' => $record['date_end'] ?: null
```

Comment:
I guess the mustache templates will skip relevantPeriod if the high date is empty right?  I was thinking we'd put this in a conditional and only populate if we have a date end but I think this is taken care of in the mustache template.  So this all looks good to me.